app/customer.py

```diff
@@ -0,0 +1,76 @@
+import math
+import datetime

```

Comment:
Add a blank line between dfferent import groups to follow PPEP-8. Please, apply this change to all places